Userland/Applications/DisplaySettings/MonitorSettingsWidget.cpp

```diff
@@ -72,7 +79,6 @@ ErrorOr<void> MonitorSettingsWidget::create_resolution_list()
 
     return {};
 }
-
```

Comment:
Would be nice if this line break remains

Reply:
Ah, right, so all functions are split apart by a newline, whoops. Should I make a new commit or just amend a previous one?

Reply:
Thank you, commit amended and newline added (hopefully did everything right). Should I resolve the conversation?

Reply:
Looks like the line is still being removed. Did you run `git push --force-with-lease` after amending locally?